fiftyone/operators/delegated.py

```
@@ -516,21 +608,172 @@ def execute_operation(
                     operation.operator,
                 )
 
-            result = asyncio.run(self._execute_operator(operation))
+            if monitor:
+                return self._execute_operation_multi_proc(
+                    operation, log, check_interval_seconds
+                )
+            else:
+                return self._execute_operation_sync(operation, log)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)